### PR TITLE
update release notes

### PR DESCRIPTION
**Pull request by jamesnocentini**

### CODE REVIEW

File: modules/ROOT/pages/release-notes.adoc [hunk 1]
```diff
@@ -29,38 +29,20 @@ xref:index.adoc[Read more]
...
-- {jira-url}/CBG-280[*CBG-280*] Syncgateway doesn't import all documents while importing 50M docs ( build 2.1.3-4)
-- {jira-url}/CBG-279[*CBG-279*] go-blip leaks nextFrameToSend goroutine under aborted multi-frame send
-- {jira-url}/CBG-265[*CBG-265*] Post Upgrade fails to remove 2.0 era design docs
-- {jira-url}/CBG-264[*CBG-264*] Post Upgrade "fails" in a cluster with no query nodes
-- {jira-url}/CBG-253[*CBG-253*] Unused bulk_docs reserved sequences may not be released
-- {jira-url}/CBG-225[*CBG-225*] Unresponsive REST API on startup
-- {jira-url}/CBG-221[*CBG-221*] Admin/User views cannot handle both xattr and non-xattr simultaneously
-- {jira-url}/CBG-189[*CBG-189*] Compact() shouldn't trigger import errors
-- {jira-url}/CBG-159[*CBG-159*] CouchbaseServerVersion should use clusterCompatibility version
+- {jira-url}/CBL-415[*CBL-415*] strod produces locale-dependent results for encoding Fleece
```
Comment (bbrks) [inline modules/ROOT/pages/release-notes.adoc]: Looks like the script was run on the wrong project. `CBL` -> `CBG`

**Review by bbrks — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/release-notes.adoc (hunk 1 above)
Comment (jamesnocentini) [reply modules/ROOT/pages/release-notes.adoc]: Oops, yes

Comment (jamesnocentini) [reply modules/ROOT/pages/release-notes.adoc]: updated.

**Comment by jamesnocentini** [comment]
@bbrks is the filter update correcting the output successfully?

**Review by bbrks — APPROVED** [review]
Hey, looks OK now! Thanks

**Comment by jamesnocentini** [comment]
@bbrks Many thanks! :)

**Pull request merged**